array-destructuring/exercise-1/exercise.js

```diff
@@ -4,10 +4,12 @@ const personOne = {
   favouriteFood: "Spinach",
 };
 
-function introduceYourself(___________________________) {
+let { name, age, favouriteFood } = personOne;
+
+function introduceYourself(personOne) {
```

Comment:
I think this would be a good time to introduce a new way to use destructuring. This function takes a parameter that is an object. We know the object will be a person that has a name, age, and favourite food.

Because we know exactly what information we need from the object, we can destructure it right here as we define the function.

`function introduceYourself({ name, age, favouriteFood })` will give us the `name`, `age`, and `favouriteFood` to use inside of the function block.

Every time we pass a person object to the function, the code will now magically get those properties for us without any extra work! 🚀 

Let me know if that makes sense @areebsattar

Reply:
Thank you for this explanation, it does make it clearer and I believe what we were doing with props in React also made it clearer as well.